application/testing/CMakeLists.txt

```diff
@@ -452,11 +453,13 @@ f3d_test(NAME TestInteractionCheatsheetWhiteBG DATA cow.vtp ARGS --background-co
 f3d_test(NAME TestInteractionCheatsheetBlackBG DATA cow.vtp ARGS --background-color=0,0,0 INTERACTION UI) #H
 f3d_test(NAME TestInteractionCheatsheetNoFile INTERACTION UI NO_DATA_FORCE_RENDER) #HXM
 f3d_test(NAME TestInteractionCheatsheetScalars DATA dragon.vtu ARGS --scalar-coloring --coloring-component=-2 INTERACTION UI LONG_TIMEOUT) #HSSS
-f3d_test(NAME TestInteractionCheatsheetOpacity DATA cow.vtp INTERACTION UI ARGS --opacity=0.5 RESOLUTION 300,700) #H;ScrollDown
 f3d_test(NAME TestInteractionCheatsheetAnimationName DATA InterpolationTest.glb ARGS --animation-index=6 INTERACTION UI) #HWWW
 f3d_test(NAME TestInteractionCheatsheetConfigFile DATA dragon.vtu CONFIG ${F3D_SOURCE_DIR}/testing/configs/bindings.json INTERACTION UI) #H;ScrollDown
 
-if(NOT F3D_MODULE_RAYTRACING)
+if(F3D_MODULE_RAYTRACING)
+  # Raytracing commands are visible in the cheatsheet
+  f3d_test(NAME TestInteractionCheatsheetSSAAOpacity DATA cow.vtp INTERACTION UI ARGS --opacity=0.5 RESOLUTION 300,700) #H;A;A;ScrollDown
```

Comment:
Can we avoid this ? I worked hard to avoid this some time ago :)

Reply:
The point of the previous test was to show the cheatsheet + model with opacity. I used scrolldown to ensure raytracing option were not visible. What changed here ?

Reply:
I added the check of `ssaa` on top of `opacity`